tests/Functional/TranslationTestBase.php

```diff
@@ -36,6 +36,7 @@ class TranslationTestBase extends BrowserTestBase {
     'options',
     'oe_translation',
     'oe_translation_test',
+    'oe_translation_metatag_test',
     'metatag',
```

Comment:
As I understand 'metatag' should be removed from the list of modules in the test.

Reply:
No not from the test.